Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.yml

```diff
@@ -443,6 +440,9 @@ script:
     - contextPath: MsGraphFiles.ListDrives.OdataContext
       description: The OData query.
       type: String
+    - contextPath: MsGraphFiles.ListDrives.NextToken
+      description: The token for the next page.
+      type: String
   - arguments:
     - auto: PREDEFINED
       description: The MS Graph resource. Can be "drives", "groups", "sites", or "users".
```

Comment:
Need to remove the following from each description that has it. Since it is a predefined list, when you generate the README, this information appears twice.  
- Can be "drives", "groups", "sites", or "users".

Reply:
Done